CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+- fix: ReactNavigationV4Instrumentation null when evaluating 'state.routes' #1940
+
+## 3.2.6
```

Comment:
Did we not merge the tag into `main`? This can happen if craft deploy failed the `github` step.
We should do that (even if manually) instead of adding the version manually here

Reply:
no, same as https://github.com/getsentry/sentry-react-native/pull/1939#discussion_r763888598